Assets/MirageXR/Managers/Activity/ActivityManager.cs

```diff
@@ -284,7 +285,7 @@ private async Task ActivateAction(string id)
                     }
 
                     await ActivateAction(action);
-                    TutorialManager.Instance.InvokeEvent(TutorialManager.TutorialEvent.ACTION_STEP_ACTIVATED);
```

Comment:
There is a direct lib-lee event for that: LearningExperienceEngine.EventManager.OnStepActivated += ACTION_STEP_ACTIVATED

Reply:
Artefact of an earlier version. It's no longer present